Live/Classes/HUD/HUDMessage.cpp

```diff
@@ -7,19 +7,19 @@ HUDMessage::HUDMessage(const std::string& message) {
     m_message = message;
 
     // Set size and position of this component
-    setPosition(Vec2(Director::getInstance()->getVisibleSize().width/2 - MSG_WIDTH/2, 60));
+    setPosition(Vec2(Director::getInstance()->getVisibleSize().width/2 - MSG_WIDTH/2, 70));
     setContentSize(Size(MSG_WIDTH,100));
 
     // Create sprite as background
-    Sprite* bg = Sprite::create("messageBackground.png");
-    bg->setPosition(Vec2(MSG_WIDTH/2, 60));
-    bg->setOpacity(100);
+    Sprite* bg = Sprite::create("spr_messaging_background.png");
+    bg->setPosition(Vec2(MSG_WIDTH/2, 63));
+    bg->setOpacity(200);
     this->addChild(bg, 0);
 
     // Create, customize, and add label
-    m_label = Label::createWithTTF(message, "fonts/emulogic.ttf", 22);
+    m_label = Label::createWithTTF(message, "fonts/emulogic.ttf", 14);
     m_label->setColor(Color3B(0, 0, 0));
-    m_label->setPosition(Vec2(MSG_WIDTH/2, 30));
+    m_label->setPosition(Vec2(MSG_WIDTH/2, 60));
     this->addChild(m_label, 1);
```

Comment:
Put the constants in macros, with reasonable documentation of what the values are used for.